local-install.sh

```diff
@@ -61,7 +61,7 @@ kubectl apply -f - <<-EOF
     metadata:
       name: back-stack
     spec:
-      package: ghcr.io/back-stack/showcase-configuration:v0.5.0
+      package: ghcr.io/back-stack/showcase-configuration:latest
```

Comment:
I figure for the purpose of the showcase, `latest` is reasonable to use. But we should recommend that users set this explicitly for real-world use.